app/models/vm_or_template.rb

```diff
@@ -128,6 +128,7 @@ class VmOrTemplate < ApplicationRecord
   has_many                  :direct_services, :through => :service_resources, :source => :service
   belongs_to                :tenant
   has_many                  :connected_shares, -> { where(:resource_type => "VmOrTemplate") }, :foreign_key => :resource_id, :class_name => "Share"
+  has_many                  :labels, -> { where(:section => "labels") }, :class_name => "CustomAttribute", :as => :resource, :dependent => :destroy
```

Comment:
@Fryguy "naming is hard" ... At first glance, I was fine with `labels`, but then I'm not sure if this should be `provider_labels`, or something indicating that these labels are imported directly from the provider.

Any thoughts here?